#This .yaml file describe the variables related to the perception process,
# during the first steps of the application

yolo_model: "yolo"
confidence_threshold : 0.35
distance_recheck : 0.4
yolo_path : "/iri_lab/iri_ws/src/cpu/Models/YOLO_Model/best_0.7.7m_e500.pt"
mobile_sam_path : "/iri_lab/iri_ws/src/cpu/Models/MOBILE-SAM/mobile_sam.pt"
detection_model : "yolo"
camera_used : "kinova"
subset_knn : 11
knn_neighbours : 3
z_angles: [-15, 0, 15]
z_rotation_angles: [-25, -15, 0, 15, 25]
max_range: 2.5 #Top threshold for maximun distance (Usage for more lightweigth octomap)